src/lib/components/friends/Friend.svelte

```diff
@@ -11,10 +11,16 @@
 </script>
 
 <div class="friend" data-cy="friend-{friend.name}">
-    <ProfilePicture id={friend.key} hook="friend-profile-picture" size={Size.Small} image={friend.profile.photo.image} status={friend.profile.status} />
-    <Text hook="friend-name" class="username" singleLine>
-        {friend.name}
-    </Text>
+    <ProfilePicture id={friend.key} hook="friend-profile-picture" size={Size.Small} loading={friend.loading} image={friend.profile.photo.image} status={friend.profile.status} />
+    <div class="friend-name-container">
+        {#if friend.loading}
```

Comment:
Let's change this to apply the loading as a prop to text, it already supports this functionality. Simply pass loading={friend.loading} on the text below